src/cunumeric/unary/scalar_unary_red_template.inl

```diff
@@ -66,68 +66,6 @@ struct ScalarUnaryRedImpl {
   }
 };
 
-template <VariantKind KIND>
-struct ScalarUnaryRedImpl<KIND, UnaryRedCode::ALL> {
-  template <LegateTypeCode CODE, int DIM>
-  void operator()(ScalarUnaryRedArgs& args) const
-  {
-    using OP    = UnaryRedOp<UnaryRedCode::PROD, LegateTypeCode::BOOL_LT>;
-    using LG_OP = typename OP::OP;
-    using VAL   = legate_type_of<CODE>;
-
-    auto rect = args.in.shape<DIM>();
-
-    Pitches<DIM - 1> pitches;
-    size_t volume = pitches.flatten(rect);
-
-    if (0 == volume) return;
-
-    auto out = args.out.reduce_accessor<LG_OP, true, 1>();
-    auto in  = args.in.read_accessor<VAL, DIM>(rect);
-
-#ifndef LEGION_BOUNDS_CHECKS
-    // Check to see if this is dense or not
-    bool dense = in.accessor.is_dense_row_major(rect);
-#else
-    // No dense execution if we're doing bounds checks
-    bool dense = false;
-#endif
-
-    ScalarUnaryRedImplBody<KIND, UnaryRedCode::ALL, CODE, DIM>()(out, in, rect, pitches, dense);
-  }
-};
-
-template <VariantKind KIND>
-struct ScalarUnaryRedImpl<KIND, UnaryRedCode::ANY> {
-  template <LegateTypeCode CODE, int DIM>
-  void operator()(ScalarUnaryRedArgs& args) const
-  {
-    using OP    = UnaryRedOp<UnaryRedCode::SUM, LegateTypeCode::BOOL_LT>;
-    using LG_OP = typename OP::OP;
-    using VAL   = legate_type_of<CODE>;
-
-    auto rect = args.in.shape<DIM>();
-
-    Pitches<DIM - 1> pitches;
-    size_t volume = pitches.flatten(rect);
-
-    if (0 == volume) return;
-
-    auto out = args.out.reduce_accessor<LG_OP, true, 1>();
-    auto in  = args.in.read_accessor<VAL, DIM>(rect);
-
-#ifndef LEGION_BOUNDS_CHECKS
-    // Check to see if this is dense or not
-    bool dense = in.accessor.is_dense_row_major(rect);
-#else
-    // No dense execution if we're doing bounds checks
-    bool dense = false;
-#endif
-
-    ScalarUnaryRedImplBody<KIND, UnaryRedCode::ANY, CODE, DIM>()(out, in, rect, pitches, dense);
-  }
-};
-
 template <VariantKind KIND>
 struct ScalarUnaryRedImpl<KIND, UnaryRedCode::CONTAINS> {
   template <LegateTypeCode CODE, int DIM>
```

Comment:
Use `RHS` instead of `VAL` in this function too?

Reply:
868b68b7adafeab1060ae5f3cffacb0b980207f2